### PR TITLE
control

### PR DESCRIPTION
**Pull request by cryogenian**
Adding control and couple event operations

@beckyconning please take a look

### CODE REVIEW

**Comment by cryogenian** [comment]
Seems that @beckyconning is very busy, @garyb maybe you can take a look at this?

File: bower.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "purescript-leaflet",
+  "name": "purescript-leafletjs",
```
Comment (garyb) [inline bower.json:2]: Was this intentional? :)

File: src/Leaflet/Core/Layer.js [hunk 2]
```diff
@@ -166,12 +172,23 @@ exports.rectangle_ = function(latlngs, converter, conf) {
 
 exports.on_ = function(e, cb, l) {
     return function() {
-        l.on(e, function() {
+        l.on(e, function(ev) {
+            console.log("in on");
+            console.log(ev);
```
Comment (garyb) [inline src/Leaflet/Core/Layer.js]: `console.log`

**Review by garyb — CHANGES_REQUESTED** [review]

File: bower.json (hunk 1 above)
Comment (garyb) [reply bower.json:2]: Oh, I see from the other PR it was. We should perhaps rename this repo to match the bower name then?

Comment (cryogenian) [reply bower.json:2]: :+1:

**Comment by cryogenian** [comment]
+ Removed `console.log`
+ Renamed the repo

**Review by garyb — APPROVED** [review]

**Pull request merged**